cloudformation/neo4jhacluster.yaml

```diff
@@ -158,7 +132,7 @@ Parameters:
   TagSystemCode:
     Description: Tag detail for the SystemCode
     Type: String
-    Default: UPP Neo4j HA
+    Default: upp
```

Comment:
Is this really the system code? I would expect upp-neo4j or upp-neo4j-cluster? What did you put in Dewey?